### ISSUE TITLE
updated and now just get white screen (no chance to even login)

### ISSUE THREAD

**Issue by petunia101** [lurume84/bling-desktop#27]
Hello,
I installed new BlingSetup.exe and now I after I click on bling desktop shortcut I just get a white screen and it doesn't even let me login...  

I tried uninstalling and rebooting and reinstalling, but still just the white screen.

I am attaching latest crash report files.
Thanks for any help you can offer as to what may be going wrong...

[8419152f-ecee-4941-b867-f6334f136132.zip](https://github.com/lurume84/bling-desktop/files/4526638/8419152f-ecee-4941-b867-f6334f136132.zip)

**Comment by lurume84** [comment]
Please send Bling.log inside Documents\Bling.exe\Logs.

Do not worry, I will release a new version today for solving this.

**Comment by lurume84** [comment]
I have released a new version that includes more troubleshooting info. Please download 0.1.17 version and check again. It may crash again but at least the dump generated will probably give me the reason why.

In the mean time, try to manually remove the viewer so the app downloads again doing the following:

1. Close the application
2. Delete zips inside %userprofile%\Documents\Bling.exe\Download\Versions\Viewer
3. Delete %userprofile%\Documents\Bling.exe\Html\viewer
4. Open the application

**Comment by petunia101** [comment]
Hello,

I uninstalled, deleted the zip files and html/viewer folder as you suggested.  Then reinstalled.  First it looked like it was going better because actually said "loading" and said something about new update or upgrade being available and did i want to load it (version 17).  i said ok and then it just seemed to get hung up again.  It generated a new crash report file so attaching that as well as the log,,,
[Bling.log](https://github.com/lurume84/bling-desktop/files/4529548/Bling.log)
[bc5a8feb-26da-481d-b53d-13455e0763ae.zip](https://github.com/lurume84/bling-desktop/files/4529553/bc5a8feb-26da-481d-b53d-13455e0763ae.zip)

Thank you.

**Comment by mikeho1218** [comment]
Seeing the same issues...
[f5bc4639-226a-4444-839f-60cfd6ffeb7c.zip](https://github.com/lurume84/bling-desktop/files/4529967/f5bc4639-226a-4444-839f-60cfd6ffeb7c.zip)

Bling.log is empty in my Logs folder

**Comment by lurume84** [comment]
Thanks for the help! Now I see what happens, it's actually the process of creating logs, included in new version. I will release a version in a few minutes with one correction. In case it does not solve your problem I will tell you how to disable logs in the ini file (I'm about to create it as it cannot be done now).

**Comment by lurume84** [comment]
I have released 0.1.18, please install it. In case application hangs again create a Bling.ini file in %userprofile%/Documents/Bling.exe with

```
[Log]
Enabled=false
```

Or, copy there the one I attach here and rename to Bling.ini.
[Bling.txt](https://github.com/lurume84/bling-desktop/files/4530440/Bling.txt)

This will disable logs so luckily you will be able to use it while I investigate the root cause. If everything works without having to disable logs, please confirm me how many log messages appear in Bling.log as the following:

`Browser instance created`

Seems that, for some reason, more than one browser instance was created in your case and I need to investigate further. 

Thanks for your contribution!

**Comment by petunia101** [comment]
hello, rename your Bling.txt file to BlinG.ini, not BlinK.ini?

I tried it with BlinG.ini but no luck, then noticed my old ini file was BlinK.ini.  going to retry as Blink.ini now and see if that makes any difference....

**Comment by lurume84** [comment]
There are two ini files Blink.ini and Bling.ini. the first one for things related with Blink servers and the second one to the actual app. But remember this is plan B, first install 0.1.18 and let's see if it crashes. If you did so delete Bling.log inside logs and open app again, if it crashes and Bling.log file is generated It means bling.ini is not being detected.

**Comment by mikeho1218** [comment]
Installed and still seeing issues... Here are the two scenarios I have tried so far:

Scenario 1: Uninstall Bling and delete Bling.exe from Documents.
Install latest Bling MSI file and start. White empty window displays and after a few seconds, Bling upgrade dialog is displayed. Click on No and login screen is displayed.
Login and application starts up. Close the application.
Bling.log and Blink.ini files are created. Ini file contains [SyncVideo] entry. I add the [Log] entry and save the file.

Restart the application and empty window just sits there. Upgrade dialog displays and click No. Application hangs.

Original Bling.log renamed to Bling.1.log and empty Bling.log file is created.

Scenario 2 After Uninstall & reinstall again, answer Yes to Bling upgrade and the application crashes and Crash Report log is created. 

Hope this helps a little. See attached files...

Also the Bling.ini file is changed and looks like this:
[Log]Enabled=false[SyncVideo]LastUpdate=2020-01-29T13:10:18+00:00

[15963095-0e1e-4e0a-b126-81cdaa1cf793.zip](https://github.com/lurume84/bling-desktop/files/4530891/15963095-0e1e-4e0a-b126-81cdaa1cf793.zip)

![BlingDlg](https://user-images.githubusercontent.com/64272739/80246233-aea83480-8631-11ea-9d36-9865578fa2df.jpg)

![BlingDlg2](https://user-images.githubusercontent.com/64272739/80246272-bec01400-8631-11ea-9091-03447c63767e.jpg)

[Bling.1.log](https://github.com/lurume84/bling-desktop/files/4530899/Bling.1.log)

**Comment by lurume84** [comment]
Thanks! It really helps. I see you place [Log] info in Blink.ini and not Bling.ini. They are two config files with different purposes. Create Bling.ini in Bling.exe folder and open again. I see you app crashes presumably when instantiating logs so the disabling had no effect. The reason why appears again the message of Blink 0.1.18 to download is because it was manually downloaded and not automatically, you can dismiss it.

Let's see if creating Bling.ini and setting Log to disabled let's you at least use the app.

**Comment by lurume84** [comment]
Oh, another thing, check on task manager there is no bling application running. I suspect this is the reason for this crash

**Comment by mikeho1218** [comment]
One of these days I will follow directions properly...

Adding [Log] to Bling.ini did not help. Application hangs on second start after closing. I have Process Explorer up and Bling is not running after closing.

I deleted all files and folder from Bling.exe folder, except for Bling.ini and restarted. After a brief delay, it started up again. I closed and interestingly there is no Logs folder. Again restarting the app leaves it hung.

**Comment by mikeho1218** [comment]
So it will start up the first time when there is no Bling.exe folder under Documents. But any other attempts to run and the app hangs.

**Comment by petunia101** [comment]
here is what i did:
Uninstalled the application
Deleted the “Bling.exe” folder under Documents (want a fresh start)
Download and run BlingSetup.exe file
Run Bling.exe – which starts and asks if want to upgrade to version 18.  I say yes and it crashes.  
There is a crash report (attached files) and under Bling.exe/Logs there is a Bling.log file but 0 kb and empty.
Checked task manager - no Bling application running
Now I am going to go add the Bling.ini file, delete the Bling.log file and try rerunning to see what happens.....
[2aa5313d-1789-4f1b-984f-a61e69283ea1.zip](https://github.com/lurume84/bling-desktop/files/4531083/2aa5313d-1789-4f1b-984f-a61e69283ea1.zip)
[CrashRpt-Log-20200424-151227-{2aa5313d-1789-4f1b-984f-a61e69283ea1}.txt](https://github.com/lurume84/bling-desktop/files/4531087/CrashRpt-Log-20200424-151227-.2aa5313d-1789-4f1b-984f-a61e69283ea1.txt)

**Comment by petunia101** [comment]
just noticed there is no Blink.ini file in Blink.exe folder (not sure if there is supposed to be one, just FYI)

**Comment by lurume84** [comment]
Ok we are almost there! Please do the following:

In Bling.ini file set 

[UpgradeDesktop]
Host=localhost

You are having crashes while trying to execute the latest Desktop installer. This will prevent automatic downloads until I discover the reason. Please tell me if this solves the problem.

**Comment by mikeho1218** [comment]
After about a six second delay on the white screen, it started up.

**Comment by lurume84** [comment]
So, are you able to use the application as before? The delay may be related with downloading latest viewer.

**Comment by mikeho1218** [comment]
Yes sir!

The captured videos are sorted strange, but that's for another time...

**Comment by lurume84** [comment]
Great! Thanks for your patience you both paid the price of being betatesters of this version. I have reproduced the issue on my machine and will release a new version today. Just in case you want to have automatic desktop updates running you will have to manually install it and remove Bling.ini. I suggest enabling logs as will help in case you get any error in the future.

I have also seen this issue related with strange ordering but it's not always happening. I think it has to do with sending requests for all video pages asynchronously so order is not being respected. 

Will take a look now that I know other people experience this.

**Comment by petunia101** [comment]
after revising the Bling.ini file to add the [UpgradeDesktop] Host=localhost it is now working for me too!! Yay!! Thank you so much for this application and your super fast fixes :)

**Comment by petunia101** [comment]
it doesn't seem to be automatically download videos though.  I've tried putting an earlier date in the .ini file but doesn't seem to make a difference.  I will close down the program (just click the x) and then reopen and the program (and it will reopen without me having to login by the way), and as soon as it opens it will download ONE video.  I wait a few minutes to see if any other videos download, but nothing.  So I close the program, and reopen (no login required) and BAM as soon as it opens, another video appears in the Download folder, but then no more videos download after that one..... until i close and reopen the program that is (then one more downloads)....
I've tried changing the date in the .ini file to an earlier date but the videos that are getting downloaded one at a time when i start the program are all from the 24th (and the ini file date changes back to the 24th as well).......  
Don't know if this is because of the sorting issue mentioned......
